files/init/start.sh

```diff
@@ -11,7 +11,7 @@ sudo python3 /etc/hobby-hub/program_manager.py /etc/hobby-hub/test_programs/ > /
 
 #Runs Flask Backend Server on startup
 echo " || Starting Web Server ||"
-python /etc/hobby-hub/web/backend/start-server.py
+# python /etc/hobby-hub/web/backend/start-server.py
```

Comment:
why? where are we starting the server then?

Reply:
forgot to remove xD